### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by drohit-cb**
### What changed? Why?

I noticed that some of the functionality I added in `v0.1.0` wasn't showing up in the corresponding docs. 

Running `npm run docs` produces the files updated/added in this PR.

### CODE REVIEW

File: docs/classes/coinbase_transaction.Transaction.html [hunk 1]
```diff
@@ -13,33 +13,36 @@
 <a href="coinbase_transaction.Transaction.html#isTerminalState" class="tsd-index-link"><svg class="tsd-kind-icon" viewBox="0 0 24 24"><use href="../assets/icons.svg#icon-2048"></use></svg><span>is<wbr/>Terminal<wbr/>State</span></a>
 <a href="coinbase_transaction.Transaction.html#rawTransaction" class="tsd-index-link"><svg class="tsd-kind-icon" viewBox="0 0 24 24"><use href="../assets/icons.svg#icon-2048"></use></svg><span>raw<wbr/>Transaction</span></a>
 <a href="coinbase_transaction.Transaction.html#sign" class="tsd-index-link"><svg class="tsd-kind-icon" viewBox="0 0 24 24"><use href="../assets/icons.svg#icon-2048"></use></svg><span>sign</span></a>
+<a href="coinbase_transaction.Transaction.html#toAddressId" class="tsd-index-link"><svg class="tsd-kind-icon" viewBox="0 0 24 24"><use href="../assets/icons.svg#icon-2048"></use></svg><span>to<wbr/>Address<wbr/>Id</span></a>
```
Comment (drohit-cb) [inline docs/classes/coinbase_transaction.Transaction.html:16]: This is the one I was hoping to see in the published docs.

**Review by alex-stone — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for alex-stone @ 2024-08-26 20:27:55 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

**Comment by cb-heimdall** [comment]
This Pull Request was merged without enough reviews
Please go [here](https://github.com/coinbase/coinbase-sdk-nodejs/issues/185) to review and +1

**Comment by cb-heimdall** [comment]
This Pull Request was merged without enough reviews
Please go [here](https://github.com/coinbase/coinbase-sdk-nodejs/issues/185) to review and +1

**Comment by cb-heimdall** [comment]
This Pull Request was merged without enough reviews
Please go [here](https://github.com/coinbase/coinbase-sdk-nodejs/issues/185) to review and +1

**Pull request merged**